src/esm_runscripts/filedicts.py

```diff
@@ -158,6 +191,42 @@ def allowed_to_be_missing(self):
         """
         return self.get("allowed_to_be_missing", False)
 
+    @property
+    def datestamp_method(self):
+        """
+        Defines which datestamp_method shall be used when possibly including
+        date stamps to the file. Valid choices are "never", "always",
+        "avoid_overwrite".
+        """
+        datestamp_method = self.get(
+            "datestamp_method", "avoid_overwrite"
+        )  # This is the old default behaviour
+        return datestamp_method
+
+    @property
+    def datestamp_format(self):
+        """
+        Defines which datestamp_format shall be used when possibly including
+        date stamps to the file. Valid choices are "check_from_filename" and
+        "append".
+        """
+        datestamp_format = self.get(
+            "datestamp_format", "append"
+        )  # This is the old default behaviour
+        return datestamp_format
+
+    def update(self, *args, **kwargs):
```

Comment:
Can we move this up, close the `__setitem__` method? Then we have all the modified `dict` methods together.

Reply:
Sure. My organisation was double underscore methods, properties, regular methods without underscore, and then private methods with one underscore. But I don't really care.